src/content/self-hosted/install/backup-restore.mdx

```diff
@@ -5,46 +5,50 @@ id: backup
 sidebar_label: Backup and Restore
 ---
 
-Backup and restore planning is essential to ensure the availability and integrity of your Okteto instance in case of unexpected failures or disaster recovery.
-This guide outlines the steps to recover an Okteto instance in such scenarios.
+This guide outlines the steps to recover an Okteto instance in the event of an unexpected failure and for disaster recovery preparedness.
 
-## Critical Resources
+## Okteto Metadata
 
-To effectively recover an Okteto instance, focus on the following critical resources:
+The most important resources to backup so you can effectively recover an Okteto instance are:
 
-- **Service Accounts**: Preserve service accounts associated with users. These service accounts reside within the `okteto` namespace and are labeled with `dev.okteto.com: "true"`.
-- **Secrets**: Recreate all secrets within the `okteto` namespace.
-- **Custom Resources**: Restore Custom Resources (CRs) corresponding to Okteto's CRDs. They can be identified with the label `app.kubernetes.io/part-of: okteto`.
+- **Service Accounts**: Okteto creates a Kubernetes service account for every user that logins into Okteto. Backup all the service accounts in the `okteto` namespace with the label `dev.okteto.com: "true"`.
+- **Secrets**: backup all the secrets in the `okteto` namespace. Okteto uses secrets to store Okteto Secrets, tokens, certificates, ssh keypairs, ...
+- **Custom Resources**: backup Custom Resources in the `okteto` namespace corresponding to the Okteto's CRDs. Okteto CRDs can be identified with the label `app.kubernetes.io/part-of: okteto`. Okteto uses CRDs to store your Catalog, Container Registry Credentials, ...
```

Comment:
```suggestion
- **Custom Resources**: Any Custom Resources in the `okteto` namespace corresponding to the Okteto CRDs. These CRDs can be identified with the label `app.kubernetes.io/part-of: okteto`. Okteto uses CRDs to store your Catalog, Container Registry Credentials, and more
```

Small changes to match tone of the above two resources. Let me know if this works/you agree with the changes 🙂